randomWords.js

```diff
@@ -3,9 +3,17 @@
 const randomWords = require('random-words')
 
 function getRandomWords(wordCount) {
+  let words
+  do {
+    words = randomWords(wordCount)
+  } while (!allElemsUnique(words))
   return randomWords(wordCount)
 }
 
+function allElemsUnique(arr) {
+  return arr.length === new Set(arr).size
```

Comment:
I had a case where "sail" appeared twice for me in my required words list. Idk if that's part of the contract for the library or what, but this was easy fix for now